### PR TITLE
Fix bugs in meeting scheduler; change some buttons from text to icons…

### PR DESCRIPTION
**Pull request by timbl**
… to be more international

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,8 +37,8 @@
   "dependencies": {
     "better-simple-slideshow": "linkeddata/better-simple-slideshow#gh-pages",
     "mime-types": "^2.1.13",
-    "solid-ui": ">=0.8.0"
-  },
+   "solid-ui": "file://../../linkeddata/solid-ui"
```
Comment (melvincarvalho) [inline package.json]: should this say file?

File: schedule/schedulePane.js [hunk 2]
```diff
@@ -537,19 +545,21 @@ module.exports = {
...
+      // emailButton.textContent = 'email invitations'
       emailButton.addEventListener('click', function (e) {
-        var title = '' + (kb.any(subject, ns.dc('title')) || '')
+        var title = kb.anyValue(subject, ns.cal('summary')) ||
+          kb.anyValue(subject, ns.dc('title')) || ''
         var mailto = 'mailto:' +
           kb.each(subject, ns.sched('invitee')).map(function (who) {
             var mbox = kb.any(who, ns.foaf('mbox'))
-            return mbox ? '' + mbox : ''
+            return mbox ? mbox.uri.replace('mailto:', '') : ''
```
Comment (melvincarvalho) [inline schedule/schedulePane.js:558]: could mbox be null here?  (dont know the answer, just checking)

Comment (timbl) [reply schedule/schedulePane.js:558]: Yes, in which case the empty string will be thing added to the main string.

Comment (timbl) [reply schedule/schedulePane.js:558]: Well, at the moment the form asks for a set of email addresses.  but later if it uses a people picker it coud  end up with a set of people soek of which don't have email addresses in the  data.

**Pull request merged**